server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_4__add_odc_user_recent_browsing.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `odc_user_recent_browsing` (
```

Comment:
no odc prefix for table name, table name follow $modueName_$resourceType naming style.
e.g.   history_resource_access

Reply:
The current function is mainly to record the user's latest access record to resources, so rename the table name to history_resource_latest_access?

Reply:
last_access will be better